storybookjs/storybook
# update peerDependency

Issue:

## What I did

remove npm WARNs

```
$ npm install    
npm WARN @storybook/addon-actions@4.0.0-alpha.1 requires a peer of @storybook/addons@^3.3.0 but none is installed. You must install peer dependencies yourself.
npm WARN @storybook/addon-notes@4.0.0-alpha.1 requires a peer of @storybook/addons@^3.3.0 but none is installed. You must install peer dependencies yourself.
npm WARN @storybook/addon-options@4.0.0-alpha.1 requires a peer of @storybook/addons@^3.3.0 but none is installed. You must install peer dependencies yourself.
```

## How to test

Is this testable with Jest or Chromatic screenshots? NO

Does this need a new example in the kitchen sink apps? NO

Does this need an update to the documentation? NO

If your answer is yes to any of these, please make sure to include it in your PR.
